lib/diaspora_federation/salmon/xml_payload.rb

```diff
@@ -11,7 +11,7 @@ module Salmon
     #   </XML>
     #
     # (The +post+ element is there for historic reasons...)
-    class XmlPayload
+    module XmlPayload
```

Comment:
Because if we don't make any instances of XmlPayload, then it's not in fact a class. Module fits better then.